### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by AswathyVNair**
Preview: https://deploy-preview-4--bakehere.netlify.app/pages/wishlist.html

### CODE REVIEW

File: css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.wishlist-pg-card-container{
+    display: flex;
```
Comment (sach10-create) [inline css/wishlist.css:2]: Can be merged in a common class

**Review by sach10-create — APPROVED** [review]
Rest code looks clean

File: css/wishlist.css (hunk 1 above)
Comment (AswathyVNair) [reply css/wishlist.css:2]: Sorry, I did'nt get the meaning.

**Review by VishnuVNairCodes — APPROVED** [review]
Good to merge

**Review by itsmebipul00 — APPROVED** [review]
Great work

**Pull request merged**